src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.cpp

```diff
@@ -1976,6 +1987,8 @@ namespace UiaOperationAbstraction
 
         switch (lhsLocal->vt)
         {
+        case VT_EMPTY:

```

Comment:
I did not thought this through and I might revert the entire variant change (and remove the test for it) since being null and being empty are not exactly the same...